improved Image component document for Blitz 2.0 (#748)
blitz-js/blitzjs.com: app/pages/docs/image-component.mdx

In addition to using properties available to the `Image` component, you
can optionally configure Image Optimization for more advanced use cases

> via `blitz.config.js`. Use `next.config.js` if you are using Blitz 2.0.

Reviewer: Since this documentation is only for Blitz 2.0, we can just say:
 ```suggestion
via `next.config.js`.
```

Author: I've made the changes that you suggested `blitz.config.js` is not mentioned now and I've also removed the previous configuration example because it was an example for `blitz.config.js`.

allowed to be optimized. This is needed to ensure that external urls can't
be abused.

> Starting from Blitz 2.0 you need to make some configuration changes
> in `next.config.js` In order to add external domains into your project.

Reviewer: ```suggestion
To add external domains to your project, you need to add the `images` property to your `next.config.js` file: 
```

```js
// @ts-check
const { withBlitz } = require("@blitzjs/next")

/**
 * @type {import('@blitzjs/next').BlitzConfig}
 **/
const config = {
  images: {

>     domains: ["www.example.com"],

Reviewer: ```suggestion
    domains: ["www.example.com"], // hightlight-line
```
